pytorch_transformers/tokenization_utils.py

```diff
@@ -519,31 +519,35 @@ def _convert_token_to_id_with_added_voc(self, token):
     def _convert_token_to_id(self, token):
         raise NotImplementedError
 
-
-    def encode(self, *text, cls_token_at_end=False, double_sep_token=False, no_sep_cls_tokens=False):
+    def encode(self, text, add_special_tokens=False, *sequences):
```

Comment:
Both Python 2 and 3 are happy with this order of arguments?

Reply:
Yes, Python 2 didn't like it when `*sequences` was placed before `add_special_tokens`

Reply:
Actually no, I hadn't updated it as I should have. It's now better, along with the tests.